clearpath_sensors/launch/flir_blackfly.launch.py

```diff
@@ -65,29 +73,65 @@ def generate_launch_description():
         parameters=[parameters, {'parameter_file': param_mapping_file,}],
         output='screen',
         remappings=[
-            ('flir_blackfly/camera_info', 'color/camera_info'),
+            ('flir_blackfly/camera_info', 'camera_info'),
```

Comment:
In our API documentation camera_info is indicated to be found at `sensors/camera_#/color/camera_info` so perhaps we need to revisit that documentation.

Reply:
@hilary-luo it is namespaced above: 
```
    arg_namespace = DeclareLaunchArgument(
        'namespace',
        default_value='platform/sensors/camera_0')
```

Reply:
That namespacing doesn't bring in the `color` section, that's all I was referring to.

Reply:
Ah, yes!

Reply:
I thought about this too, the issue is that `camera_info` is published by the driver and used by all subsequent post-processing images. So, it makes sense to have it at the top level.